radar_control/radar_control.py

```diff
@@ -686,24 +688,26 @@ def radar():
                                          sm.COLOR("blue", round(bound_time_remaining, 6)))
                         rad_ctrl_print(msg)
 
-                        if bound_time_remaining < aveperiod.intt*1e-3:
+                        if bound_time_remaining < aveperiod.intt * 1e-3:
                             # reduce the integration period to only the time remaining
                             # until the next scan boundary.
+                            # TODO: Check for integration_period_done_time < 0
+                            # integration_period_done_time is in units of ms
```

Comment:
I don't think this is true; it is a datetime object.
```suggestion

```

Reply:
aaah, I was confused by the previous comments, but I see now they were for bound_time_remaining and intt in the calculation